plugins/woocommerce/src/Internal/RestApi/Routes/V4/Settings/Email/Schema/EmailSettingsSchema.php

```
@@ -0,0 +1,263 @@
+		);
+	}
+
+	/**
+	 * Get email settings data by transforming email settings into REST API format.
+	 *
+	 * @param mixed           $item             Settings array from WC_Settings_Emails.
+	 * @param WP_REST_Request $request          Request object.
+	 * @param array           $include_fields   Fields to include.
+	 * @return array
+	 */
+	public function get_item_response( $item, WP_REST_Request $request, array $include_fields = array() ): array {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)